### ISSUE TITLE
How to process images of different sizes?

### ISSUE THREAD

**Issue by Kongsea** [vanhuyz/CycleGAN-TensorFlow#65]
This repository processes the fixed-size images, however, I need to input and output images of different sizes and make sure that the images remain the same sizes as its original ones.
So can it be modified to support this need? And how to modify it?
Thank you very much.

**Comment by ngeorgis** [comment]
https://github.com/vanhuyz/CycleGAN-TensorFlow/issues/42

**Comment by Kongsea** [comment]
Thank you.
It said the input images need to be squared, however, my images are not.
How to process it or modify the code?

**Comment by ngeorgis** [comment]
Yes, seem that they need to be square images. Not sure if it is an easy modification to change the source to support non-square images. I may try later. In the meantime, maybe scaling or padding could work but that may depend on your application.

**Comment by Kongsea** [comment]
Thank you. I will try it too.

**Comment by xymeng16** [comment]
Have u guys got some results on the non-squared images? I'm working on it too.. @Kongsea @ngeorgis

**Comment by Kongsea** [comment]
Have you achieved it? @xymeng16

**Comment by xymeng16** [comment]
Well, I try to apply padding and resize the picture before the input but sadly neither showed a good performance(my application is car license plate generation, which has a fixed size and aspect ratio). So I traverse the code and find I need to modify the implementation of some layers in ops.py. I think I will open a PR after finishing it. @Kongsea

**Comment by Kongsea** [comment]
Look forward to it.

**Comment by mattchiang-gsp** [comment]
Just to be clear, is this discussion about "training" on various/bigger image sizes like 512x512 or "evaluating" a model on different sized inputs?

**Comment by Zxr1314** [comment]
@xymeng16 I try to apply the code on PR data generation,too! You can modify the code to process different W and H size plate images.

**Comment by XuJ1E** [comment]
Hello! Is there anyone solve the problem :input image size over 256 ?such as 512 or higher
